lightning/src/ln/channel.rs

```diff
@@ -1541,8 +1552,185 @@ pub(super) struct ChannelContext<SP: Deref> where SP::Target: SignerProvider {
 	/// If we can't release a [`ChannelMonitorUpdate`] until some external action completes, we
 	/// store it here and only release it to the `ChannelManager` once it asks for it.
 	blocked_monitor_updates: Vec<PendingChannelMonitorUpdate>,
+
+	/// The current interactive transaction construction session under negotiation.
+	#[cfg(any(dual_funding, splicing))]
+	interactive_tx_constructor: Option<InteractiveTxConstructor>,
+	// If we've sent `commtiment_signed` for an interactive transaction construction,
+	// but have not received `tx_signatures` we MUST set `next_funding_txid` to the
+	// txid of that interactive transaction, else we MUST NOT set it.
+	next_funding_txid: Option<Txid>,
```

Comment:
```suggestion
	#[cfg(any(dual_funding, splicing))]
	next_funding_txid: Option<Txid>,
```

Reply:
This is ok to leave without a flag as it's already in the reestablish messages as a TLV and will just be None since we never set it.